apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/FetchedGrunnlag.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.etterlatte.grunnlag
+
+import com.fasterxml.jackson.databind.JsonNode
+import no.nav.etterlatte.libs.common.grunnlag.Grunnlagsopplysning
+import no.nav.etterlatte.libs.common.person.Folkeregisteridentifikator
+
+data class FetchedGrunnlag(
```

Comment:
HentetGrunnlag*

Reply:
Ser det er litt gjennomgående engelsk her, men vi har vel strengt talt blitt enige om å bruke norsk der man ikke må bruke særengelske uttrykk for det.